modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/upgrade/v4_2_1/test/UpgradeCalendarBookingTest.java

```diff
@@ -119,6 +130,22 @@ protected void assertSameTime(
 			actualJCalendar.get(java.util.Calendar.MINUTE));
 	}
 
+	protected CalendarBooking createCalendarBooking() throws PortalException {
```

Comment:
@rijema make it private

Reply:
As I sent to an util class, I had to leave it public. Is there any problem?